packages/icons/README.md

```diff
@@ -0,0 +1,115 @@
+# ./packages/icons
+
+This package is for custom Supabase icons
+They can be used alongside any other icon packages
+
+## example use
+
+```jsx
+import { ReplaceCode, InsertCode } from 'icons'
+
+function app() {
+  return (
+    <>
+      <ReplaceCode className="text-light" strokeWidth={1} size={16} />
+      <InsertCode className="text-light" strokeWidth={1} size={16} />
+    </>
+  )
+}
+```
+
+## adding new icons
+
+Add new icons into ./src/raw-icons
```

Comment:
@MildTomato these icons should always be exported 24x24px and have an icon that's around 18x18px(ish) for consistency, right? Might be worth mentioning in the README.md

Reply:
adding

Reply:
done